### PR TITLE
Bugfix: Remove indents

### PR DESCRIPTION
**Pull request by Nedava22**
Bugfix to index_3.js

### CODE REVIEW

File: hw/hw_0001/index.3.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+  {shortname: "CAD", name: "Канадський долар", cost: 18.3881},
+  {shortname: "HRK", name: "Куна", cost: 3.6095},
+  {shortname: "MDL", name: "Молдовський лей", cost: 1.3754},
 ];
 
 console.log("Currencies:");
 
 currencies.forEach((currency) => {
-    console.log(`- ${currency.shortname} costs ${currency.cost} hrn (${currency.name})`);
+  console.log(`- ${currency.shortname} costs ${currency.cost} hrn (${currency.name})`);
```
Comment (volodymyrkr) [inline hw/hw_0001/index.3.js]: Move full currency name after shortname

File: hw/hw_0001/index.3.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 });
 
 var totalCost = currencies.reduce((prev, currency) => {
-    return prev + currency.cost;
+  return prev + currency.cost;
```
Comment (volodymyrkr) [inline hw/hw_0001/index.3.js]: rename prev to sum

**Review by volodymyrkr — CHANGES_REQUESTED** [review]

File: hw/hw_0001/index.3.js (hunk 1 above)
Comment (Nedava22) [reply hw/hw_0001/index.3.js]: Full currency name moved after shortname

File: hw/hw_0001/index.3.js (hunk 2 above)
Comment (Nedava22) [reply hw/hw_0001/index.3.js]: Prev renamed to sum

File: hw/hw_0001/index.3.js (hunk 1 above)
Comment (volodymyrkr) [reply hw/hw_0001/index.3.js]: Ok

**Review by volodymyrkr — APPROVED** [review]

**Pull request merged**